tls.go

```diff
@@ -16,6 +17,8 @@ func TLS13Config() *tls.Config {
 	}
 }
 
+// MutualTLS13Config returns *tls.Config with mTLS 1.3 settings given the
+// client set of certificates.
```

Comment:
Same as above but this will do Mutual certificate verification via the certificate authority specified.